scss/_modal.scss

```diff
@@ -176,27 +176,18 @@
 
 // Scale up the modal
 @include media-breakpoint-up(sm) {
-  // Automatically set modal's width for larger viewports
-  .modal-dialog {
+  .modal {
     --#{$prefix}modal-margin: #{$modal-dialog-margin-y-sm-up};
+    --#{$prefix}modal-box-shadow: #{$modal-content-box-shadow-sm-up};
+  }
 
+  // Automatically set modal's width for larger viewports
+  .modal-dialog {
     max-width: var(--#{$prefix}modal-width);
     margin-right: auto;
     margin-left: auto;
   }
 
-  .modal-dialog-scrollable {
-    height: calc(100% - var(--#{$prefix}modal-margin) * 2);
-  }
-
-  .modal-dialog-centered {
-    min-height: calc(100% - var(--#{$prefix}modal-margin) * 2);
-  }
```

Comment:
What's the intention behind removing these rules?

Reply:
They are already written at line 77 and 92 of the same file, so it is just removing dupe code. However, maybe the expected behavior is awaited for sm breakpoint and up ?

Reply:
Oh yeah, good catch then! <3